src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -421,6 +421,17 @@ theorem truncate_succ (x : BitVec w) :
     have j_lt : j.val < i := Nat.lt_of_le_of_ne (Nat.le_of_succ_le_succ j.isLt) j_eq
     simp [j_eq, j_lt]
 
+/-! ### concat -/
+
+@[simp] theorem getLsb_concat (x : BitVec w) (b : Bool) (i : Nat) :
+    (concat x b).getLsb i = match i with
+      | 0 => b
+      | i+1 => x.getLsb i := by
```

Comment:
I'm skeptical of this as a simp lemma with the match on the RHS.

The @[simp] lemmas should be for `getLsb_concat_zero` and `getLsb_concat_succ`.